test/download-handler.js

```diff
@@ -35,7 +35,7 @@ exports.tests = {
         ShellJS.rm(tmpPath);
 
         // Download
-        var url = "http://www.linuxfoundation.org";
+        var url = "https://www.linuxfoundation.org";
```

Comment:
Should I remove this test too, or change it to https? It's the only test in download-handler.js